pkg/apis/core/v1beta1/defaults_cloudprofile.go

```diff
@@ -50,3 +41,31 @@ func SetDefaults_VolumeType(obj *VolumeType) {
 		obj.Usable = ptr.To(true)
 	}
 }
+
+// GetV1JsonCapabilities transforms the given keys and values into a JSON-string and returns it as v1.JSON object.
```

Comment:
As this is only used within the cloudprofile tests atm, would you mind moving the function over to the corresponding file?